backend/app/database/crud.py

```diff
@@ -94,29 +102,58 @@ def get_contact_list_by_user_id(db: Session, user_id: int):
 
 
 def add_contact_list(db: Session, email: str, user_id: int, is_subscribed: bool):
-    contact = models.ContactList(email = email, user_id = user_id, is_subscribed = is_subscribed)
+    contact = models.ContactList(
+        email=email, user_id=user_id, is_subscribed=is_subscribed)
     session = Session()
     try:
         db.add(contact)
         db.commit()
     except:
         session.rollback()
-        raise 
+        raise
     finally:
         session.close()
 
 
 def delete_contact_by_email_and_user_id(db: Session, email: str, user_id: int):
     session = Session()
     try:
-        db.query(models.ContactList).filter(models.ContactList.email == email, models.ContactList.user_id == user_id).delete()
+        db.query(models.ContactList).filter(models.ContactList.email ==
+                                            email, models.ContactList.user_id == user_id).delete()
         db.commit()
     except:
         session.rollback()
         raise
     finally:
         session.close()
 
+
+def unsubscribe_contact_by_email_and_user_id(db: Session, email: str, user_id: int):
+    session = Session()
+    try:
+        db.query(models.ContactList).filter(
+            models.ContactList.email == email, models.ContactList.user_id == user_id).update({"is_subscribed": False})
+        db.commit()
+    except Exception as err:
```

Comment:
If you dont use `err` later on, you actually don't need to specify `Exception`.
Just the code like below should work.

```py
except:
    session.rollback()
    raise
```

This is where you might wanna declare err
```py
except Exception as err:
    print(err)
```

Reply:
I think you referred to my code, sorry! My latest PR shouldn't have that so forgive me~